plugins/woocommerce/client/legacy/js/admin/meta-boxes-order.js

```diff
@@ -488,6 +488,13 @@ jQuery( function ( $ ) {
 						if ( response.success ) {
 							$( '#woocommerce-order-items' ).find( '.inside' ).empty();
 							$( '#woocommerce-order-items' ).find( '.inside' ).append( response.data.html );
+
+							// Update notes.
+							if ( response.data.notes_html ) {
+								$( 'ul.order_notes' ).empty();
+								$( 'ul.order_notes' ).append( $( response.data.notes_html ).find( 'li' ) );
```

Comment:
Optional small thing ... I realize the performance benefit is small in real terms:

```suggestion
								$( 'ul.order_notes' )
									.empty()
									.append( $( response.data.notes_html ).find( 'li' ) );
```

Since most methods like these return the same jQuery object back, we can chain them and avoid the cost of re-selecting the same elements. Could apply to the similar chunk further down.